actions/user_actions.jsx

```diff
@@ -35,6 +35,18 @@ export function loadProfilesAndStatusesInChannel(channelId, page = 0, perPage =
     };
 }
 
+export function loadProfilesAndReloadTeamMembers(page, perPage, teamId, options) {
+    return async (doDispatch, doGetState) => {
+        const newTeamId = teamId || getCurrentTeamId(doGetState());
+        const {data} = await doDispatch(UserActions.getProfilesInTeam(newTeamId, page, perPage, '', options));
+        if (data) {
+            await Promise.all([doDispatch(loadTeamMembersForProfilesList(data, newTeamId, true)), doDispatch(loadStatusesForProfilesList(data))]);
```

Comment:
Yea but to do that I would have to have call the actual client functions and then dispatch the actions after receiving the data - I think its not really a big deal on web since we're doing it everywhere anyway. It would also be different than how its done everywhere else in the WebApp. I dont think the performance here is an issue

Reply:
Got it. That's a valid point. Thanks Farhan.